src/Microsoft.PowerShell.SHiPS/SHiPSConstants.cs

```diff
@@ -26,6 +26,20 @@ internal static class Constants
             "Resolve-Path",
             "Get-Item",
             "Test-Path"
+        };
+
+        // Although we don't support these commands, we do handle their error messages.
+        internal static string[] NotSupportedCommands = {
```

Comment:
should we combine Supported and Not supported commands to one providercommands?

Reply:
We can, if there is no other usage of the Supported list, now and future.